src/layout/Sidebar/index.tsx

```diff
@@ -96,17 +94,6 @@ export default function Sidebar({ collapsed, onCollapsed }: SidebarProps) {
     }
   ]
 
-  // add v1
-  if (Object.keys(allPool)?.length > 0) {
-    mainLinks.push({
-      link: `${BETA_MENU_LINK.migrate}/1`,
-      icon: Migration,
-      title: 'Migrate',
-      id: 'migrate',
-      isActive: location?.pathname?.startsWith(BETA_MENU_LINK.migrate)
-    })
-  }
```

Comment:
lets not completely remove it, instead may be lets just do comment for now and then when we make beta site default we will remove it if not needed